openlibrary/templates/type/user/view.html

```diff
@@ -37,6 +38,12 @@ <h1>$page.displayname</h1>
 
     $:sanitize(format(page.description))
 
+    <h2>Reading Log</h2>
+    $if settings.get('public_readlog') == "yes":
+      <p>This user has chosen to publicly share the books they are <a href="$(page.key)/books/currently-reading">currently reading</a>, <a href="$(page.key)/books/already-read">have already read</a>, and <a href="$(page.key)/books/want-to-read">want to read</a>!</p>
+    $else:
+      <p>This user has chosen to keep their Reading Log private</p>
```

Comment:
QUESTION: Is this a downer to articulate that someone has made their Reading Log private? Should we show nothing here? Not sure, just curious...

Reply:
Hmm, I wondered this too. It was designed this in response to @yonial's feedback -- if a user tries to go to a private reading log, they get re-routed to the user's profile page (and we show this message). Agreed it's not perfect

Reply:
Understood. I tend to think that eventually, the Reading Log should be a part of the user's profile page, shown or hidden depending on the user's settings. Anyway, fine to proceed.

Reply:
agreed, there's actually a design + path to replace this, so will keep this in consideration

Reply:
@mekarpeles I agree with @LeadSongDog 's comment on an earlier version of this text, about the subtle change to make it clear the privacy the default.  This version may not be quite accurate either, as the user has not necessarily chosen anything at this point: "This user has not chosen to share their Reading Log"  would be more accurate, or if you want less of a downer, maybe "This user has not yet chosen to share their Reading Log" . The ''yet'' may be too much / hopeful / misleading though, I don't have a strong opinion :)  I think there should be a way forward that is both more accurate reflecting default privacy (which is a good thing), and simultaneously less of a downer.